### ISSUE TITLE
Fix GitHub Actions

### ISSUE THREAD

**Issue by ronaldtse** [expresslang/course#2]
The `tex/latexmkrc` file provided works.

On macOS, I had to do the following to run:

```
brew install mactex
cd tex
latexmk
# => done
```

In the current GHA run (https://github.com/expresslang/course/actions/runs/1648367315), latexmk fails on inclusion of files like `models.1` because `mpost` was not run before the run. `mpost` is used to split the `models.mp` and `models.mpx` files into `models.1`... `models.XX`.

Somehow, this GHA action does not perform that. I have tried a few other GHA actions for latexmk but they all expect some kind of file name as input and ignores the `@default_files` variable inside latexmk.

e.g.
```
2022-01-03T07:41:35.0054647Z LaTeX Warning: File `models.1' not found on input line 201.
2022-01-03T07:41:35.0055017Z 
2022-01-03T07:41:35.0055208Z 
2022-01-03T07:41:35.0055737Z ! LaTeX Error: File `models.1' not found.
2022-01-03T07:41:35.0056049Z 
2022-01-03T07:41:35.0056527Z See the LaTeX manual or LaTeX Companion for explanation.
2022-01-03T07:41:35.0057103Z Type  H <return>  for immediate help.
2022-01-03T07:41:35.0057530Z  ...                                              
2022-01-03T07:41:35.0057866Z                                                   
2022-01-03T07:41:35.0058329Z l.201 \includegraphics{models.1}
2022-01-03T07:41:35.0058737Z                               
2022-01-03T07:41:35.0059033Z ? 
2022-01-03T07:41:35.0059371Z ! Emergency stop.
2022-01-03T07:41:35.0059727Z  ...                                              
2022-01-03T07:41:35.0060070Z                                                   
2022-01-03T07:41:35.0060491Z l.201 \includegraphics{models.1}
2022-01-03T07:41:35.0060910Z                               
```

This ticket is to fix that.

**Comment by ronaldtse** [comment]
@camobap would you have time for this? Thanks!

**Comment by ronaldtse** [comment]
Closed in #4
